checker/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta name="generator" content="HTML Tidy for HTML5 for Apple macOS version 5.7.16">
+<title>WebIDL 2 Checker</title>
+
+<script src='../lib/webidl2.js'></script>
+<script>
+let parserResult = undefined;
+
+function formatParserOutput() {
+  const outputEl = document.getElementById('webidl-checker-output');
+  if (parserResult) {
+    const prettyPrintEl = document.getElementById('pretty-print');
+    outputEl.innerText = JSON.stringify(parserResult, null, prettyPrintEl.checked ? 2 : null);
+  } else {
+    outputEl.innerText = '';
+  }
+}
+
+function checkWebIDL() {
```

Comment:
checkWebIDL() should only take a `textToCheck`.

Reply:
Done.